<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QWebEngineView" name="mapWebWidget" native="true"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="documentsFrame">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="7" column="0" colspan="2">
         <widget class="QProgressBar" name="mapProgressBar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="loadMapButton">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Show Map for this Document</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="amenitiesLabel">
          <property name="font">
           <font>
            <family>Droid Sans Fallback</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Amenities</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QListWidget" name="listaDocsWidget">
          <property name="maximumSize">
           <size>
            <width>1244</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="documentsLabel">
          <property name="font">
           <font>
            <family>Droid Sans Fallback</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Documents</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="radiusBox">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QLabel" name="documentDisplayedName">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Showing no document</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QTreeWidget" name="amenitiesList">
          <column>
           <property name="text">
            <string>Select Amenities</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
       <zorder>mapProgressBar</zorder>
       <zorder>listaDocsWidget</zorder>
       <zorder>documentDisplayedName</zorder>
       <zorder>loadMapButton</zorder>
       <zorder>documentsLabel</zorder>
       <zorder>amenitiesLabel</zorder>
       <zorder>radiusBox</zorder>
       <zorder>amenitiesList</zorder>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="leyendaFrame">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="leyendaLabel">
          <property name="font">
           <font>
            <family>Droid Sans Fallback</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Legend</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="callesList">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
